tests/unit/DateTest.php

```diff
@@ -419,4 +435,15 @@ public function it_can_be_offset_by_days(int $days): void
 
         $this->assertTrue($expectedDate->equals($offsetDate));
     }
+
+    /**
+     * @test
+     */
+    public function it_can_be_constructed_from_a_number_of_years_ago(): void
```

Comment:
Need to consider what we do with this method as it's using the current timestamp which doesn't align well with the rest of the library, consider changing this to `Date::fromYearsAgo(Clock $c, int $years): Date` potentially? Or drop it?

Reply:
Good for now; drop the method later; implement `offsetByYears` (similar to already existing `offsetByDays`); be happy :crab: